recipes/dyff/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "dyff" %}
+{% set version = "1.5.1" %}
+
+{% set goname = "github.com/homeport/" + name.lower() %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+    sha256: 933d3545451019ccb35d799a4cf09e775b973d3b0b9ecef82c16e407ae6975c4
+
+build:
+  number: 1
```

This should start from zero and only be incremented if the previous build number was uploaded to the main channel:
```suggestion
  number: 0
```

---

Roger, thanks. Ready for re-review.